// EA-72093: Empty intersection of types...
fun foo(first: Array<Any?>, second: Array<Any?>) =
        Pair(first.toCollection(<!NO_VALUE_FOR_PARAMETER!>)<!>, second.toCollection(<!NO_VALUE_FOR_PARAMETER!>)<!>)